src/Nethermind/Nethermind.Runner/Ethereum/Steps/LoadGenesisBlockAuRa.cs

```diff
@@ -0,0 +1,52 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System.Linq;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Dirichlet.Numerics;
+using Nethermind.Runner.Ethereum.Context;
+using Nethermind.Specs.Forks;
+
+namespace Nethermind.Runner.Ethereum.Steps
+{
+    public class LoadGenesisBlockAuRa : LoadGenesisBlock
```

Comment:
That would be really cool if we somehow could move AuRa steps to AuRa namespace and leave the Runner namespace clean. Not sure if possible.

Reply:
It is possible, but:
1. I would have to move base steps (all steps?) to even more general places:

StepsProject (not real name) -> Aura/Clique -> Runner

2. This would introduce even more complexity & confusion (would be too magical) and would be harder to find out what is going on during startup. I even named steps StepNameEngineName instead EngineNameStepName as even latter makes more sense its harder to spot on explorer/solution view.

We can further discuss this.